public/js/custom.js

```diff
@@ -0,0 +1,26 @@
+$(document).on('change', 'select', function () {
```

Comment:
@Evolyuta well done on the jQuery solution - smart to hide the options instead of showing validation errors